core/lib/config/src/configs/chain.rs

```diff
@@ -121,9 +121,9 @@ pub struct StateKeeperConfig {
     pub close_block_at_eth_params_percentage: f64,
     /// Denotes the percentage of L1 gas used in L2 block that triggers L2 block seal.
     pub close_block_at_gas_percentage: f64,
-
-    pub fee_account_addr: Address,
-
+    /// Fee account address. Value is deprecated and it's used only for generating wallets struct
+    #[deprecated]
+    pub fee_account_addr: Option<Address>,
```

Comment:
By the way, you can create a separate structure for deprecated one-time configs (e.g. in `zksync_env_config` crate), and deserialize them separately.
They'll have a comment explaining that these exist until we fully switch to a file-backed configuration scheme and aren't actually used by the core codebase otherwise.
This way you'll make sure that nobody actually uses these fields.

Not a strong requirement, though.

Reply:
It's already complicated, let's not overcomplicate it...